config/environments/profile.rb

```diff
@@ -0,0 +1,41 @@
+# an environment configured for performance profiling.
```

Comment:
i'm curious where you got this from. It does not exactly match what we have as environments/production.rb, any reason not to just copy (or even load) environments/production.rb, so we know we're profiling under our actual production config?